fix(dapps) restore loading of Wallet Connect SDK

stefandunca opened the pull request:
### Closes: #15694

Reverts changes introduced by #15565 the connector integration that were loading the SDK asynchronously. Not sure why it isn't working though.

### Affected areas

Wallet Connect, Dapps Connector

### How to test

Connect a wallet connect dApp or check the existing connections

### Risk 

Described potential risks and worst case scenarios.

Tick **one**:
- [x] Low risk: 2 devs MUST perform testing as specified above and attach their results as comments to this PR **before** merging.

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 47bcda2e | [#1](https://ci.status.im/job/status-desktop/job/prs/job/linux/job/x86_64/job/tests-nim/job/PR-15695/1/) | 2024-07-22 08:49:53 | ~6 min | `tests/nim` | [:page_facing_up:`log`](https://ci.status.im/job/status-desktop/job/prs/job/linux/job/x86_64/job/tests-nim/job/PR-15695/1//consoleText) |

Seitseman reviewed (approved):
Tested locally, connections to dApps is working.

kounkou commented:
> Tested locally, connections to dApps is working.

Hey @Seitseman would you mind sharing some screenshots please?

stefandunca commented:
@kounkou I just updated with the agreed chances and removed the `sdkLoader` component entirely. Please test that connector works fine and let's merge this fix.

kounkou commented:
Tested that both Wallet connect and Connector service work with current changes.
Thanks @stefandunca 

[Screencast from 2024-07-22 08-45-53.webm](https://github.com/user-attachments/assets/b2327053-f003-4652-8403-4ed46094a4fc)
